package/com.unity.formats.usd/Runtime/Scripts/IO/Skel/SkeletonImporter.cs

```diff
@@ -15,6 +15,7 @@
 using System;
 using System.Collections.Generic;
 using pxr;
+using Unity.Collections;
```

Comment:
Unity.Collections does not exist in older versions of Unity -- hence the conditional include a few lines below this.

Reply:
Done